api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto

```diff
@@ -275,6 +275,31 @@ message ExternalProcessor {
   // backend stream lifetime. In this case, Envoy will eventually timeout the external processor stream according to this time limit.
   // The default value is 5000 milliseconds (5 seconds) if not specified.
   google.protobuf.Duration deferred_close_timeout = 19;
+
+  // [#not-implemented-hide:]
+  // Enables sending body to the side stream server when the header external processing is still ongoing.
+  // It only works for STREAMED body processing mode. For any other body processing modes, it is ignored.
+  //
+  // In the use cases if the side stream server must wait for at least one chunk of body before sending back
+  // the response for the headers, this filter configuration has to be enabled to support this.
```

Comment:
```suggestion
  // Setting this flag to true is useful in cases where the side stream server must wait for at least one chunk of body before sending the response for the headers back.
```

It is unclear what does "back" refer to. Can you clarify this?

Reply:
"back to Envoy". Thanks for pointing it out.